### PR TITLE
ENH: Plot Config Import/Export

### PR DESCRIPTION
**Pull request by aksharsarvesh**
Builds on #27

### CODE REVIEW

File: archive_viewer/mixins/plot_config.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+from widgets import ColorButton
+from pyqtgraph import ViewBox
+from typing import Dict
+from qtpy.QtGui import QColor
+from time import sleep
+
+class PlotConfigMixin:
```
Comment (YektaY) [inline archive_viewer/mixins/plot_config.py:7]: Style: Can you add docstrings to the class and all the methods in this class? thanks

**Review by YektaY — APPROVED** [review]
This looks great. Only noted for more docstrings for one of the classes.

**Comment by aksharsarvesh** [comment]
Docstrings is a common issue of mine, apologies. Check again?

**Comment by aksharsarvesh** [comment]
Dude, I hate merge conflicts so much

File: archive_viewer/mixins/plot_config.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 from widgets import ColorButton
 from pyqtgraph import ViewBox
-
+from typing import Dict
+from qtpy.QtGui import QColor
+from time import sleep
```
Comment (zdomke) [inline archive_viewer/mixins/plot_config.py:5]: Style: Reorder imports and remove the unused import. I typically follow this order:

1. full library imports (`import re`)
2. standard library imports (`from os import path`)
3. qtpy imports (`from qtpy.QtCore import Qt`)
4. pyqtgrah imports
5. pydm imports (`from pydm.widgets import PyDMPushButton`)
6. local module imports (`from config import logger`)

When I have multiple imports that match a category, I sort them by library name length. e.g.:
```
from qtpy.QtGui import QColor, QBrush, QPen
from qtpy.QtWidgets import QPushButton
```

All that to say I'd order these imports like this:
```
from typing import Dict
from qtpy.QtGui import QColor
from pyqtgraph import ViewBox
from widgets import ColorButton
```

**Review by zdomke — COMMENTED** [review]
This looks great. I just had one style suggestion for how I order my imports block

**Review by YektaY — APPROVED** [review]

**Pull request merged**